slink/docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: slink_web_1
+      APP_PORT: 3000
+
+  web:
+    image: anirdev/slink:v1.7.3@sha256:4dc9e1849a0a814d51c031f1e3870b9c58767192b3fff4f46a89005879ca0042
+    restart: on-failure
+    environment:
+      TZ: Europe/Berlin
+      ORIGIN: http://umbrel.local:8811
```

`umbrel.local` relies on mDNS resolution, which may not always be available or properly configured in all environments or containers.

Using `localhost` ensures the app resolves correctly inside Docker and avoids DNS or network resolution issues with `.local` domains.

```suggestion
      ORIGIN: http://localhost:8811
```

---

Hey @Mr-Sunglasses,

Great review! 👏 

Best practice here is to use the `DEVICE_DOMAIN_NAME` environment variable. This ensures that the right hostname is being used, so like this:
`ORIGIN: http://${DEVICE_DOMAIN_NAME}:8811`

Otherwise creating an account in the app would fail.